core/domain/config_domain_test.py

```diff
@@ -24,33 +24,36 @@
 from core.domain import config_domain
 from core.platform import models
 from core.tests import test_utils
+from mypy_imports import config_models
```

Comment:
This import should be after `if MYPY:` please look into other files to see how to do it.

Reply:
done